File: algaworks/fullstack_angular_spring/algamoneyapi/src/main/java/br/com/flaviogf/algamoneyapi/models/Person.java
package br.com.flaviogf.algamoneyapi.models;

import javax.persistence.*;

@Entity
@Table(name = "people")
public class Person {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;
    private String name;

    @Embedded
    private Address address;
    private Boolean active;

    public Long getId() {
        return id;
    }

    public void setId(Long id) {
        this.id = id;
    }

    public Address getAddress() {
        return address;
    }

    public void setAddress(Address address) {
        this.address = address;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public Boolean getActive() {
        return active;
    }

    public void setActive(Boolean active) {
        this.active = active;
    }
}

File: alura/php_oo_002/bank/src/Models/Person.php
<?php

namespace Bank\Models;

class Person
{
    private string $name;
    private Document $document;

    public function __construct(string $name, Document $document)
    {
        $this->name = $name;
        $this->document = $document;
    }

    public function getName(): string
    {
        return $this->name;
    }

    public function getDocument(): string
    {
        return $this->document->getValue();
    }
}